include/cpr/accept_encoding.h

```diff
@@ -0,0 +1,41 @@
+#ifndef CPR_ACCEPT_ENCODING_H
+#define CPR_ACCEPT_ENCODING_H
+
+#include <curl/curlver.h>
+#include <initializer_list>
+#include <map>
+#include <string>
+#include <vector>
+
+namespace cpr {
+
+enum class AcceptEncodingMethods {
+  identity,
+  deflate, 
+  zlib, 
+  gzip,
+};
+
+class AcceptEncoding {
+  public:
+    std::map<AcceptEncodingMethods, std::string> MethodsStringMap { 
+      {AcceptEncodingMethods::identity, "identity"}, 
+      {AcceptEncodingMethods::deflate, "deflate"}, 
+      {AcceptEncodingMethods::zlib, "zlib"}, 
+      {AcceptEncodingMethods::gzip, "gzip"} 
+    };
+
+    AcceptEncoding() = default;
+    AcceptEncoding(const std::initializer_list<AcceptEncodingMethods>& methods);
+    AcceptEncoding(const std::initializer_list<std::string>& methods);
+
+    bool has() const;
```

Comment:
I thought about it again and I prefer `empty()` here as name.

Reply:
Sure!